cmd/cloud-controller-manager/app/controllermanager.go

```diff
@@ -68,13 +51,15 @@ the cloud specific control loops shipped with Kubernetes.`,
 			verflag.PrintAndExitIfRequested()
 			cliflag.PrintFlags(cmd.Flags())
 
-			c, err := s.Config(KnownControllers(), ControllersDisabledByDefault.List())
+			c, err := s.Config(knownControllers(), app.ControllersDisabledByDefault.List())
```

Comment:
app.ControllersDisabledByDefault.List() is probably not what we want to do. This is deferring responsibility for the set of controllers to be deferred to K/K rather then gcp. A concrete example is we may want to disable on of the controllers when we turn on import NodeIPAMController. If we ever end up switching to our own ServiceController, then we will want to disable the default one.  So this is fine for now but we should plan to switch to a list we control here.

Reply:
Those will be addressed in v1.21.  Thanks